docs/spec/data-plane.md

```diff
@@ -91,6 +91,25 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### HTTP Attributes
+
+An event sender, including Source and Broker and Channel, MUST include
```

Comment:
Can't be a MUST. This is a breaking change.

Reply:
Done